lesson02/assignment/index.html

```diff
@@ -5,11 +5,45 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
         <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 02</title>
+        <title>Assignment 02 - Profile Card</title>
+        <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
     </head>
     <body>
-        <main class="main">
-            <h1>title</h1>
+        <main class="profile-card-container">
```

Comment:
When I think of a semantic HTML element used to represent a card, `main` doesn't immediately come to mind. Maybe consider keeping `main` a simple container with padding/margin.

And then maybe the card itself is a `section`, while the elements inside the card are `div`s or something different.

All this is a bit subjective, and I'm sorta nitpicking. This might be a helpful read: https://developer.mozilla.org/en-US/docs/Glossary/Semantics